[Question] Can I develop the project in Windows?

### 描述
win10系统，icu库和c++编译器已经下载完毕，icu库已移动至C:\icu目录下。但使用uv安装依赖的时候还是报错了，请问这些问题该怎么解决？
```
PS D:\ChenJiaQi\Github\Research\ragflow-main2>  uv sync --python 3.10 --all-extras # install RAGFlow dependent python modules
Resolved 378 packages in 1ms
  × Failed to build `pyicu==2.14`
  ├─▶ The build backend returned an error
  ╰─▶ Call to `setuptools.build_meta.build_wheel` failed (exit code: 1)

      [stdout]

      Building PyICU 2.14 for ICU 75.1 (max ICU major version supported: 76)

      running bdist_wheel
      running build
      running build_py
      copying py\icu\__init__.py -> build\lib.win-amd64-cpython-310\icu
      running build_ext
      building 'icu._icu_' extension
      D:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX86\x64\cl.exe /c /nologo /O2 /W3 /GL /DNDEBUG /MD -Ic:/icu/include
      -IC:\Users\Administrator\AppData\Local\uv\cache\builds-v0\.tmp8mOgE9\include -IC:\Users\Administrator\AppData\Roaming\uv\python\cpython-3.10.16-windows-x86_64-none\include
      -IC:\Users\Administrator\AppData\Roaming\uv\python\cpython-3.10.16-windows-x86_64-none\Include -ID:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include
      -ID:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Tools\MSVC\14.42.34433\ATLMFC\include -ID:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Auxiliary\VS\include "-IC:\Program Files (x86)\Windows      
      Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files    
      (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /EHsc    
      /Tp_icu_.cpp /Fobuild\temp.win-amd64-cpython-310\Release\_icu_.obj /Zc:wchar_t /EHsc -DPYICU_VER=\"2.14\" -DPYICU_ICU_MAX_VER=\"76\"
      _icu_.cpp
c:/icu/include\unicode/localpointer.h(561): error C3533: 参数不能为包含“auto”的类型
c:/icu/include\unicode/localpointer.h(561): note: 非类型模板参数中的“auto”至少需要“/std:c++17”
c:/icu/include\unicode/uenum.h(69): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ucnv.h(597): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/locid.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/uloc.h(708): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/ures.h(268): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ucal.h(803): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uformattable.h(109): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/fmtable.h(39): error C2429: 语言功能 "嵌套命名空间定义" 需要编译器标志 "/std:c++17"
c:/icu/include\unicode/ufieldpositer.h(83): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/unumberoptions.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/unum.h(460): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/udat.h(1022): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/numfmt.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/measunit.h(36): error C2429: 语言功能 "嵌套命名空间定义" 需要编译器标志 "/std:c++17"
c:/icu/include\unicode/stringoptions.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/uchar.h(3176): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/utext.h(1596): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ubrk.h(372): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/udatpg.h(201): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/upluralrules.h(136): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/decimfmt.h(63): error C2429: 语言功能 "嵌套命名空间定义" 需要编译器标志 "/std:c++17"
c:/icu/include\unicode/udata.h(434): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uset.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/uset.h(358): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/unorm2.h(286): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ucol.h(553): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uniset.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/uniset.h(714): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/ucsdet.h(109): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uidna.h(169): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/translit.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/utrans.h(258): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ulocdata.h(118): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uregex.h(229): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/usearch.h(376): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ucasemap.h(99): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ubidi.h(579): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uformattedvalue.h(436): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uspoof.h(1579): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uspoof.h(1597): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/uldnames.h(97): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ulistformatter.h(236): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ulistformatter.h(247): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ureldatefmt.h(315): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/ureldatefmt.h(326): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/simpleformatter.h(28): error C2429: 语言功能 "嵌套命名空间定义" 需要编译器标志 "/std:c++17"
c:/icu/include\unicode/ubiditransform.h(320): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/edits.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/numberformatter.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/uformattednumber.h(217): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/unumberformatter.h(570): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/numberformatter.h(96): error C2429: 语言功能 "嵌套命名空间定义" 需要编译器标志 "/std:c++17"
c:/icu/include\unicode/numberformatter.h(1692): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/unumberrangeformatter.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/unumberrangeformatter.h(448): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/unumberrangeformatter.h(465): error C2975: “closeFunction”:“icu_75::internal::LocalOpenPointer”的模板参数无效，应为编译时常量表达式
c:/icu/include\unicode/localpointer.h(561): note: 参见“closeFunction”的声明
c:/icu/include\unicode/localebuilder.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
c:/icu/include\unicode/localematcher.h(1): warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失
D:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include\optional(11): warning STL4038: The contents of <optional> are available only with C++17 or later.
c:/icu/include\unicode/localematcher.h(683): error C2039: "optional": 不是 "std" 的成员
D:\ChenJiaQi\VisualStudio2022\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include\atomic(272): note: 参见“std”的声明
c:/icu/include\unicode/localematcher.h(683): error C2143: 语法错误: 缺少“;”(在“<”的前面)
c:/icu/include\unicode/localematcher.h(683): error C4430: 缺少类型说明符 - 假定为 int。注意: C++ 不支持默认 int
c:/icu/include\unicode/localematcher.h(683): error C2238: 意外的标记位于“;”之前
error: command 'D:\\ChenJiaQi\\VisualStudio2022\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.42.34433\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2

      hint: This usually indicates a problem with the package or the build environment.
  help: `pyicu` (v2.14) was included because `ragflow` (v0.15.1) depends on `pyicu`
```

--- a/deepdoc/vision/operators.py
+++ b/deepdoc/vision/operators.py
@@ -145,18 +145,6 @@ class ToCHWImage(object):
         return data
 
 
-class Fasttext(object):
-    def __init__(self, path="None", **kwargs):
-        import fasttext
-        self.fast_model = fasttext.load_model(path)
-
-    def __call__(self, data):
-        label = data['label']
-        fast_label = self.fast_model[label]
-        data['fast_label'] = fast_label
-        return data
-
-
 class KeepKeys(object):
     def __init__(self, keep_keys, **kwargs):
         self.keep_keys = keep_keys

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,7 +36,6 @@ dependencies = [
     "elastic-transport==8.12.0",
     "elasticsearch==8.12.1",
     "elasticsearch-dsl==8.12.0",
-    "fasttext==0.9.3",
     "filelock==3.15.4",
     "flask==3.0.3",
     "flask-cors==5.0.0",

